big_tests/test.config

```diff
@@ -269,7 +269,7 @@
       {mod_roster, "[modules.mod_roster]
   backend = \"rdbms\""}]},
     {mysql_redis,
-     [{dbs, [redis, mysql]},
+     [{dbs, [redis, mysql, rmq]},
```

Comment:
I think CircleCI just starts Rabbit for all presets (see `config.yml`. Did you check what happens to it after this change? My guess is that it would be started twice and it would be better to remove it from `config.yml`, right?

Reply:
it won't.  we use this list to set up DB env for GH Actions in automatic way.
CircleCI sets it explicitly for different presents..